component/src/main/java/org/wso2/siddhi/runner/beam/SiddhiAppContainer.java

```
@@ -0,0 +1,287 @@
+            String stream = "define stream " + streamName + " (event object);";
+            this.streamDefinitions.add(stream);
+            this.transformsMap.put(SiddhiAppContainer.generateTransformName(transform.getFullName()), transform);
+            this.collectionsMap.put(SiddhiAppContainer.generateTransformName(transform.getFullName()), keyCollection);
+
+            /*
+            Create queries for each transform mapped by output collections
+             */
+            if (transform.getOutputs().isEmpty()) {
+                if (transform.getTransform() instanceof TextIO.Write) {
+                    String sinkType = "text";
+                    String sinkStreamName = "textSinkStream"
+                            + (multiSinkcount == 1 ? "" : String.valueOf(multiSinkcount));
+                    try {
+                        TextIO.TypedWrite textio = ((TextIO.Write) transform.getTransform()).withOutputFilenames();
+                        Class<?> cls = textio.getClass();
+                        Method getFileNamePrefix = cls.getDeclaredMethod("getFilenamePrefix");
+                        getFileNamePrefix.setAccessible(true);
+                        ValueProvider.StaticValueProvider provider =
+                                (ValueProvider.StaticValueProvider) getFileNamePrefix.invoke(textio);
+                        String filePath = provider.get().toString();
+                        this.queryDefinitions.add(generateSinkQuery(sinkType, streamName, sinkStreamName));
+                        this.streamDefinitions.add(generateSinkStream(sinkType, sinkStreamName, filePath));
+                    } catch (NoSuchMethodException | SecurityException exception) {
+                        log.error(exception.getMessage(), exception);
+                    } catch (Exception exception) {
+                        log.error(exception.getMessage(), exception);
```

Why are we swallowing the exception here. Add context aware message.